### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Education Data Migration, Yellow Ribbon Tool

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#99571]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I really appreciate the effort that's gone into this since the design intent! Thank you!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Coding note:** As the coding for the [cards](https://www.figma.com/design/uk70as0p979ECDolNVSaMQ/Design-for-YR-in-CT?node-id=455-38517&t=hIMGtYlzXProPRyf-4) is finalized, keyboard focus management is really important here. When content changes on the page as a result of some JS action, those content changes aren't immediately announced to screen reader users unless you do something to ensure that there's an announcement. There are a few different ways to do that, but the one that's typically preferred on VA.gov is to send keyboard focus to a useful place. When focus is assigned to something, that thing gets announced immediately.
   
   For the "Display results" button, I would expect that after pressing that button focus goes to the "Showing 1 - 3 of X..." text that appears above the cards. Screen reader users will hear the "Showing..." message, which gives them a confirmation that the content of the page has changed and a summary of how it's changed. Likewise, when clicking a pagination link (eg. going from 1 to 2), would expect focus to go to the same place: "Showing 4 - 6 of X..." for the same reasons.
- [x] **Must/Coding note:** Thinking about how a screen reader user would experience these [cards](https://www.figma.com/design/uk70as0p979ECDolNVSaMQ/Design-for-YR-in-CT?node-id=455-38517&t=hIMGtYlzXProPRyf-4), it may be hard to discern where one card ends and the next begins just from the text (and without the visual cues). I think either of these two options would be fine:
   - Include some kind of unique heading for each card. (Note that if necessary for design reasons you can visually hide headings using the `sr-only` class.) This is what we typically see on VA.gov, but should only be done if you can reliably provide a unique heading based on the content you have.
   - Code the cards as an unordered list `ul`, where each card is a `li`. This introduces list semantics into the cards and would ensure that screen readers announce each card as a discrete item with an index, which should make it easier to identify the boundaries between cards. Note that there's a known issue with VoiceOver on MacOS when the bullet point styling is removed from a `ul` list that can be resolved using `role="list"` --- so you'll end up with something like:

```
<ul role="list">
   <li>
      <va-card ... />
   </li>
   <li>
      <va-card ... />
   </li>
   ...
</ul>
```

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by cassidy-beach** [comment]
**Coding note:** As the coding for the [cards](https://www.figma.com/design/uk70as0p979ECDolNVSaMQ/Design-for-YR-in-CT?node-id=455-38517&t=hIMGtYlzXProPRyf-4) is finalized, keyboard focus management is really important here.
**Must/Coding note:** Thinking about how a screen reader user would experience these [cards](https://www.figma.com/design/uk70as0p979ECDolNVSaMQ/Design-for-YR-in-CT?node-id=455-38517&t=hIMGtYlzXProPRyf-4)
EDM addresses both of these in EDM-505: Review coding for screen readers on Collab Cycle ticket per Midpoint Review. Ticket completed 12/30/2024..
